src/Profile/Attribute/DocumentDetails.php

```diff
@@ -117,13 +115,33 @@ public function getIssuingAuthority(): ?string
     private function parseFromValue(string $value): void
     {
         $parsedValues = explode(' ', $value);
+        $this->validateParsedValues($parsedValues);
+
         $this->setType($parsedValues);
         $this->setIssuingCountry($parsedValues);
         $this->setDocumentNumber($parsedValues);
         $this->setExpirationDate($parsedValues);
         $this->setIssuingAuthority($parsedValues);
     }
 
+    /**
+     * @param string[] $parsedValues
+     *
+     * @return void
+     */
+    private function validateParsedValues(array $parsedValues): void
+    {
+        if (count($parsedValues) < 3) {
+            throw new AttributeException('Invalid value for DocumentDetails');
+        }
+
+        foreach ($parsedValues as $parsedValue) {
+            if (strlen($parsedValue) === 0) {
+                throw new AttributeException('DocumentDetails cannot have empty values');
```

Comment:
In this scenario I don't know if this would make much sense to have "empty values", as this could just be an extra space in a OCR-read value. 
I've just put "Invalid Document Details":
```suggestion
                throw new AttributeException('Invalid DocumentDetails value');
```
That might make more sense, what do you think?

Reply:
Rebased and fixed in ef55028